spec/models/game_spec.rb

```diff
@@ -43,4 +43,47 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      expect(game.pieces.count).to eq 32
+    end
+
+    it "should successfully hook up the database relationship between pieces and games" do
```

Comment:
This isn't actually part of `populate_board!`, so it shouldn't be tested in this `describe` block. In fact, you can drop this test entirely, since I'll be adding specs for these associations as part of https://trello.com/c/Srt3LB1I/32-create-connections-between-the-user-game-and-piece-models, which is almost ready.